src/ResourceManager/TrafficManager/Commands.TrafficManager2.Test/SessionRecords/Microsoft.Azure.Commands.TrafficManager.Test.ScenarioTests.EndpointTests/TestCreateExistingEndpoint.json

```diff
@@ -1,8 +1,8 @@
 {
```

Comment:
That's quite strange, I have this in my csproj instead of explicit includes for individual files. I'll try CopyAlways, but PreserveNewest should work just the same, unless there is some bad datetime handling between git, the build machine and my GMT+00 timezone 

    <None Include="SessionRecords\**\*.json">
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
    </None>